LedgerSMB/Sysconfig.pm

```diff
@@ -98,9 +101,13 @@ our $no_db_str = 'database';
 our $log_level = 'ERROR';
 our $DBI_TRACE=0;
 # available printers
-our %printer;
+our %printer = ();
 
-my $cfg = Config::IniFiles->new( -file => "ledgersmb.conf" ) || die @Config::IniFiles::errors;
+my $cfg;
+# Try Loading the config file
+eval { $cfg = Config::IniFiles->new( -allowempty => 1, -reloadwarn => 1, -file => "ledgersmb.conf" ); }
+# Fall back to an empty default config
+if ( ! $cfg ); $cfg=new Config::IniFiles( );
```

Comment:
@sbts that's correct. the $cfg=... piece of code isn't linked to the 'if' statement, because that statement was terminated by the semi-colon before the '$cfg='. So, it's an unconditional statement.